docs/docs/reference/posteriors.md

```diff
@@ -0,0 +1,26 @@
+# Posteriors
+
+::: sbi.inference.posteriors.direct_posterior.DirectPosterior
+    selection:
+      filters: [ "!^_", "^__", "!^__class__" ]
```

Comment:
I have a question on these filters. Do they work for main page as well as for the table of contents (right side)? In other words? is it possible to filter different things for them?

Reply:
I don't think this works to have different navigation on the right side as I think this is mostly decided based on the headers part and not based on `mkdocstrings` extension which is the one responsible for generating this part.

But I am pretty sure there are some options to tweak the the navigation bar in `mkdocs` and to filter some of the headers in the TOC.

Reply:
I see. Can you explain to my why the __init__ fcns are displayed in the doc in the last snapshot picture (2 days ago) but not in the one before that? (search for "API rendering after change:")?

Reply:
because pedro changed the heading levels.
basically the nav shows all headerd above level 3, my proposal was with class and func as header 3 (and thus methods as header 4) while now it is header 2.

Reply:
I see, so the lower `heading_level` is, the  more (less important) things are displayed

Reply:
I see, so the lower `heading_level` is, the  more (less important) things are displayed